ledger/ledgercore/statedelta.go

```diff
@@ -190,10 +195,8 @@ func MakeNewAccountDeltas(hint int) NewAccountDeltas {
 		accts:      make([]NewBalanceRecord, 0, hint*2),
 		acctsCache: make(map[basics.Address]int, hint*2),
 
-		appParamsCache:      make(map[AccountApp]int),
-		appLocalStatesCache: make(map[AccountApp]int),
-		assetParamsCache:    make(map[AccountAsset]int),
-		assetHoldingsCache:  make(map[AccountAsset]int),
+		appResourcesCache:   make(map[AccountApp]int),
+		assetResourcesCache: make(map[AccountAsset]int),
```

Comment:
maybe initialize the `assetResourcesCache`/`appResourcesCache` with the size of the `hint` ?
most of the performance cost is when you get to the point where you start copying data.

Reply:
OK!